website/views/pages/pricing.ejs

```diff
@@ -163,9 +163,8 @@
                     <div class="col-6 col-md-8 d-flex">
                       <img
                         alt="a bullet point ellipse"
-                        src="/images/ellipse-6x8@2x.png"
-                        class="mt-2"
-                        style="width: 6.4px; height: 8px"
+                        src="/images/ellipse-6x6@2x.png"
+                        style="width: 6px; height: 6px; margin-top: 0.6rem"
```

Comment:
might consider using px for the margins here instead of rem (I find it to be a tad more predictable that way)

Reply:
Ok, I will convert this to 9.6px